medical_prescription_state_verify/data/medical_prescription_order_state_data.xml

```diff
@@ -1,72 +1,23 @@
-        <record id="state_verification"
-                model="medical.prescription.order.state">
-            <field name="name">Verification</field>
-            <field name="description">
-                Indicates that the Prescription has not yet been verified.
-            </field>
-            <field name="type">normal</field>
-            <field name="sequence">10</field>
-            <field name="fold">False</field>
-        </record>
-        
-        <record id="state_exception"
-                model="medical.prescription.order.state">
-            <field name="name">Exception</field>
-            <field name="description">
-                Indicates that the Prescription has an issue that needs to be manually resolved.
-            </field>
-            <field name="type">exception</field>
-            <field name="sequence">20</field>
-            <field name="fold">False</field>
-        </record>
-        
-        <record id="state_verified"
-                model="medical.prescription.order.state">
-            <field name="name">Verified</field>
-            <field name="description">
-                Indicates that the Prescription has been verified, but additional manual action
-                is required.
-            </field>
-            <field name="type">verified</field>
-            <field name="sequence">30</field>
-            <field name="fold">False</field>
-        </record>
-        
-        <record id="state_complete"
-                model="medical.prescription.order.state">
-            <field name="name">Complete</field>
-            <field name="description">
-                Indicates that the Prescription has been verified, and all initial manual action
-                has been completed.
-            </field>
-            <field name="type">verified</field>
-            <field name="sequence">40</field>
-            <field name="fold">True</field>
-        </record>
-        
-        <record id="state_cancel"
-                model="medical.prescription.order.state">
-            <field name="name">Cancelled</field>
-            <field name="description">
-                Indicates that the Prescription was cancelled.
-            </field>
-            <field name="type">cancel</field>
-            <field name="sequence">50</field>
-            <field name="fold">True</field>
-        </record>
-        
-    </data>
+    <record id="prescription_order_state_exception" model="base.kanban.stage">
+        <field name="name">Exception</field>
+        <field name="description">Indicates that the Prescription has an issue that needs to be manually resolved.</field>
+        <field name="sequence" eval="6" />
+        <field name="fold" eval="False" />
+        <field name="res_model_id" ref="medical_prescription.model_medical_prescription_order" />
+    </record>
+
+    <record id="prescription_order_line_state_exception" model="base.kanban.stage">
+        <field name="name">Exception</field>
```

Comment:
It should also be sequenced between Ordered and PV2, not between PV2 and Complete